### PR TITLE
preferUnplugged: true

### PR DESCRIPTION
**Pull request by johnrichardrinehart**
TL;DR: Using `yarn` version `berry` seems incompatible with package `open` using default PnP settings.

# Context

I ran into a problem using [`storybook`](https://www.learnstorybook.com/). `storybook` starts a webserver on a certain port using a library which calls `open`. I'm using `yarn` version `berry` (`v2.4.0`) which stores all packages in zipped form (PnP). This seems to be incompatible with the behavior of spawning a new child process with `open`. I obtained the following traceback:

```
➜  charm-prospector git:(john/cleanup) ✗ yarn storybook
info @storybook/react v6.1.11
info 
info => Using prebuilt manager
info => Loading presets
info => Loading 1 config file in "./.storybook"
info => Loading 7 other files in "./.storybook"
info => Adding stories defined in ".storybook/main.js"
info => Using default Webpack setup
webpack built 9cb17c8972c4de8382bf in 6707ms
╭────────────────────────────────────────────────╮
│                                                │
│   Storybook 6.1.11 started                     │
│   8.3 s for preview                            │
│                                                │
│    Local:            http://localhost:6006/    │
│    On your network:  http://10.0.2.15:6006/    │
│                                                │
╰────────────────────────────────────────────────╯
node:internal/child_process:403
    throw errnoException(err, 'spawn');
          ^

Error: spawn ENOTDIR
    at ChildProcess.spawn (node:internal/child_process:403:11)
    at Object.spawn (node:child_process:573:9)
    at module.exports (/home/john/Downloads/repos/charm-prospector/.yarn/cache/open-npm-7.3.0-d582d8b42e-a212fc9cf6.zip/node_modules/open/index.js:152:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:93:5) {
  errno: -20,
  code: 'ENOTDIR',
  syscall: 'spawn'
}
```

This seems to be related to https://github.com/yarnpkg/berry/issues/856.

I've managed to fix this problem by adding `preferUnplugged: true` to your `package.json`. When I instruct `yarn` to use my fork with this fix everything works fine:

```package.json
  ....
  "resolutions": {
    "open": "johnrichardrinehart/open#john/prefer-unplugged"
  }
```

```
➜  charm-prospector git:(john/cleanup) ✗ yarn storybook
info @storybook/react v6.1.11
info 
info => Using prebuilt manager
info => Loading presets
info => Loading 1 config file in "./.storybook"
info => Loading 7 other files in "./.storybook"
info => Adding stories defined in ".storybook/main.js"
info => Using default Webpack setup
webpack built 9cb17c8972c4de8382bf in 8392ms
╭────────────────────────────────────────────────╮
│                                                │
│   Storybook 6.1.11 started                     │
│   9.64 s for preview                           │
│                                                │
│    Local:            http://localhost:6006/    │
│    On your network:  http://10.0.2.15:6006/    │
│                                                │
╰────────────────────────────────────────────────╯
  
```

No error, here.

### CODE REVIEW

**Comment by johnrichardrinehart** [comment]
I just saw that this package author is opposed to `yarn`-specific fields added to `package.json`. This seems a bit unreasonable to me, since all of `package.json` is somewhat tool-specific and it's only a single-line change.

Anyone who's having this problem can specify my repository as a dependency by adding:

```json
  "resolutions": {
    "open": "johnrichardrinehart/open#john/prefer-unplugged"
  }
```
to their `package.json`.

**Comment by merceyz** [comment]
Already tried to get this merged in https://github.com/sindresorhus/open/pull/185 but @sindresorhus refused to merge it, which is a bit interesting as in https://github.com/sindresorhus/open/issues/169#issuecomment-580583850 they said 
> They are the ones breaking the whole ecosystem. They should also be the one to fix problems.

**Comment by johnrichardrinehart** [comment]
@merceyz You're right. It seems to me there are 2 options:

1. `yarn install && @merceyz's command && yarn run <command>` with https://github.com/sindresorhus/open
2. `yarn install && yarn run <command>` with https://github.com/johnrichardrinehart/open#prefer-unpugged

**Comment by merceyz** [comment]
You only need to run unplug once, yarn will keep it unplugged "forever" after that

**Comment by johnrichardrinehart** [comment]
@merceyz Yep. That's true. It's another step in the project set-up for new developers. That's all I'm saying.

**Comment by johnrichardrinehart** [comment]
@merceyz I just read https://github.com/yarnpkg/berry/issues/856 . I understand what you mean, now.

He's right. `yarn unplug -AR open` and commit your modified `package.json`. I'm deleting my forked repo, now.
